x-pack/plugins/security_solution/public/management/components/hooks/use_url_pagination.ts

```diff
@@ -64,6 +64,20 @@ export const useUrlPagination = (): UrlPagination => {
   const location = useLocation();
   const history = useHistory();
   const { urlParams, toUrlParams } = useUrlParams();
+
+  useEffect(() => {
+    // Replace old page_index and page_size url params by the new ones, page and pageSize
+    if ((urlParams.page_index && !urlParams.page) || (urlParams.page_size && !urlParams.pageSize)) {
+      history.replace(
+        `${history.location.pathname}${location.search
+          .replaceAll('page_size', 'pageSize')
+          .replaceAll('page_index', 'page')}`
+      );
+    }
+    // Apply this only when on mount this hook
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
🤢  😆 

can we avoid this eslint disable rule?

Reply:
Is there a way to have an empty hook deps without a eslint error? I'm trying to run this only when component mounts

Reply:
maybe pair it with `isMounted()` and just allow it run on MOunt?

Another option is to use a `useRef()` and set the `reference.current` to a function, then use that here. but it is weir :)